libs/libcommon/src/libcommon/queue.py

```diff
@@ -168,6 +177,7 @@ class JobDocument(Document):
     namespace = StringField(required=True)
     priority = EnumField(Priority, default=Priority.LOW)
     status = EnumField(Status, default=Status.WAITING)
+    difficulty = IntField(required=True)
```

Comment:
Maybe as default, it could be 50?

Reply:
Hmmm, I wanted to force the field to be set when creating a job. What do you think?

Reply:
Sure, I was thinking to set a default to avoid doing many modifications in tests for example but it is ok for me.